genai-perf/genai_perf/parser.py

```diff
@@ -604,6 +647,14 @@ def _add_endpoint_args(parser):
         help="URL of the endpoint to target for benchmarking.",
     )
 
+    endpoint_group.add_argument(
+        "--server-metrics-url",
+        type=str,
+        default=None,
+        required=False,
+        help="URL of the server metrics endpoint. Required for 'openai' service kind. Defaults to the default URL if 'service_kind' is 'triton'.",
```

Comment:
This doesn't seem to be required. It'd be good to avoid adding required args.

Reply:
I will modify the comment.